packages/antd/package.json

```
@@ -0,0 +1,79 @@
+{
+  "name": "@react-jsonschema-form/antd",
```

```suggestion
  "name": "@rjsf/antd",
```

This is the name under which the package will be published. Can you rename to the above? Once this PR is merged, I can publish it under the `rjsf` npm organization.

---

Also please replace other occurrences in the readme.

---

Just to double-check, will the core package remain `react-jsonschema-form`, or will that also migrate to the `rjsf` npm organization?

---

I'm leaning more towards keeping it called `react-jsonschema-form`. It would be easier to upgrade that way; and that way, if someone installs the `@latest` version of it, they won't get the v2.0.0-alpha.1 alpha version.

---

Hey, just to let you know, I've updated this -- changed the name back to `@rjsf/core` for the core package -- it seemed best to keep all rjsf packages under the `@rjsf` organization to maintain a clear separation between org-owned packages and other community packages. We can still keep this called `@rjsf/antd` though.